tests/char_suffer_test.cpp

```diff
@@ -167,14 +168,15 @@ TEST_CASE( "suffering from albinism", "[char][suffer][albino]" )
             WHEN( "not wearing sunglasses" ) {
                 REQUIRE_FALSE( dummy.worn_with_flag( flag_SUN_GLASSES ) );
 
-                THEN( "they suffer about 1 pain per hour" ) {
+                // 60 times * 1 bodyparts * 0.1 chance for severe effect
+                THEN( "they suffer about 6 pain per hour" ) {
                     test_suffer( dummy, 1_hours );
-                    // This will pass if pain is between 0 and 6 in an hour
-                    CHECK( dummy.get_pain() == Approx( 1 ).margin( 5 ) );
+                    CHECK( dummy.get_pain() == Approx( 6 ).margin( 4 ) );
```

Comment:
You ammm you failed your own pain test. You can increase the range of the test or revise/adjust your pain numbers.

Reply:
Hmm, that worked for me often enough. I've increased the the time to 3 hours to get better values. That should resolve that issue.